src/main/java/com/todolist/app/servlet/NewTaskServlet.java

```diff
@@ -0,0 +1,52 @@
+package com.todolist.app.servlet;
+
+
+import com.todolist.app.model.Item;
+import com.todolist.app.model.Priority;
+import com.todolist.app.model.Status;
+import com.todolist.app.model.Tag;
+import com.todolist.app.service.ToDoStoreSingleton;
+import jakarta.servlet.*;
+import jakarta.servlet.http.*;
+import jakarta.servlet.annotation.*;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+
+@WebServlet(name = "NewTaskServlet", value = "/new-task")
+public class NewTaskServlet extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        RequestDispatcher requestDispatcher = request.getRequestDispatcher("/jsp/new-task.jsp");
+        requestDispatcher.forward(request, response);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        ToDoStoreSingleton singleton = ToDoStoreSingleton.getInstance();
+        String uuid = singleton.addItem(getItem(request));
+        response.sendRedirect(request.getContextPath() + "/notification/?operation=ADD&id=" + uuid);
```

Comment:
Let's remove that slash at the end of notification `/notification/` path. See https://en.wikipedia.org/wiki/Query_string